recipes/centos_spring_boot/Dockerfile

```diff
@@ -8,7 +8,7 @@ FROM registry.centos.org/che-stacks/centos-jdk8
 MAINTAINER Gytis Trikleris
 
 EXPOSE 8080
-LABEL che:server:8080:ref=springboot che:server:8080:protocol=http
+LABEL che:server:8080:ref=springboot che:server:8080:protocol=http 
```

Comment:
extra space should be removed

Reply:
will be sending another PR for child images deriving out of it, will apply these comments